doc/ref_cert/RC2/chapters/chapter02.md

```diff
@@ -139,6 +139,33 @@ The following software versions are considered to benchmark Kubernetes v1.20
 | Functest                | leguer      |
 | xrally-kubernetes       | 1.1.1.dev12 |
 
+### SIG Testing
+
+The Reference Conformance suites must be stable and can be executed on real
+deployments. Then all the following labels are defacto skipped in
+[End-to-End Testing](https://github.com/kubernetes/community/blob/master/contributors/devel/sig-testing/e2e-tests.md):
+  - Disruptive
+  - Flaky
+  - alpha
+
+[End-to-End Testing](https://github.com/kubernetes/community/blob/master/contributors/devel/sig-testing/e2e-tests.md)
+basically respectivelly asks focus and fix regexes to select or to blacklist
```

Comment:
small typo - respectivelly -> respectively

I don't know if its only me but I don't see the option of adding a "suggestion" commit anymore.

Reply:
thx. see new commit.